survey-service/src/main/java/pl/hycom/surveyservice/controller/EditSurveyController.java

```diff
@@ -0,0 +1,51 @@
+package pl.hycom.surveyservice.controller;
+
+import org.bson.types.ObjectId;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpRequest;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import pl.hycom.surveyservice.model.Page;
+import pl.hycom.surveyservice.model.Question;
+import pl.hycom.surveyservice.model.Survey;
+import pl.hycom.surveyservice.repository.SurveyRepository;
+
+import javax.validation.Valid;
+import java.util.List;
+import java.util.Optional;
+
+@RestController
+@CrossOrigin(origins = "*", allowedHeaders = "*")
```

Comment:
Ogólnie * w mechanizmie  cross origin jest lekko niebezpieczny, lepiej zapinać się na konkretne domeny.
Nawet z jakieś konfiguracji. Nie jest to wielki "błąd" tylko taka uwag ana przyszłość.